### PR TITLE
Partner Manager 2.11 with a rewritten Activity Tracking topic

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/activity-tracking.adoc [hunk 1]
```diff
@@ -1,87 +1,129 @@
...
+. Search for the transmission.
 +
-If you have not received an acknowledgment from your trading partner or if your trading partner sent a rejection acknowledgment, then you can review the acknowledgment status of outbound transactions.
-* Transformation and routing information
-+
-An exclamation point (!) appearing next to any of the rectangle boxes in the *Transmission Flow* section indicates an issue with the transmission.
-. Click *View Payload* to view and download the originally received payload.
+When you search for the transission, you can filter the results based on the following criteria:
+
+* Message type, such as `my-po-ack:ORD-9892330`
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: The first filter with the label "Type", is actually direction - with dropdown menu options "Receive from partners" and "Send to partners".

File: modules/ROOT/pages/activity-tracking.adoc [hunk 2]
```diff
@@ -1,87 +1,129 @@
...
+
+The *Summary* section for an inbound transmission includes the following information:
+
+* Transmission status (a green checkmark for success or a red exclamation point for failure)
+* Transmission ID
+* Name of the transmitted file
+* Control number associated with the transmitted file, if available
+* Name of the receiving (Receive from Partners) endpoint
+* Date and time when the file was received
+* Message type, such as `mythical-po-ack-ORD-9892330`
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: Message type, such as `mythical-po-ack"

The ORD-9892330 is the business Key value (such as order number)

File: modules/ROOT/pages/activity-tracking.adoc [hunk 3]
```diff
@@ -1,87 +1,129 @@
...
+
+|===
+|Action |Steps
+
+|View and download the payload received from the partner (*Vijay, is this correct? How do users see the metadata for an inbound message?*)
+a|
+. In the *Transmission Flow* section, click *View Payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN message sent to the partner
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: View and download the MDN message sent to the partner for inbound AS2 transmissions.

File: modules/ROOT/pages/activity-tracking.adoc [hunk 4]
```diff
@@ -1,87 +1,129 @@
...
+. Click *View Payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the payload sent to the partner
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the *Sent transmission* field, click *View payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN received from the partner
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: View and download the MDN received from the partner in response to outbound AS2 transmissions.

File: modules/ROOT/pages/activity-tracking.adoc [hunk 5]
```diff
@@ -1,87 +1,129 @@
...
+* Date and time when the file was received
+* Message type, such as `mythical-po-ack-ORD-9892330`
+* Number of received messages in the file
+
+Take any of the following actions to view and download additional information about the inbound transmission:
+
+|===
+|Action |Steps
+
+|View and download the payload received from the partner (*Vijay, is this correct? How do users see the metadata for an inbound message?*)
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: Yes - this is correct

File: modules/ROOT/pages/activity-tracking.adoc [hunk 6]
```diff
@@ -1,87 +1,129 @@
...
+| View and download the MDN received from the partner
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the *Received MDN* field, click *View payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+|View the CONTRL message received from the partner, if it exists
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the ??? field *Click *View payload*. (*Vijay, what is the name of this field? I only see the Waiting for CONTRL status here so I can't tell.*)
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: This is same as the experience in X12. 
In the *Sent transmission* field, click *View payload*.

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/activity-tracking.adoc [hunk 7]
```diff
@@ -1,87 +1,130 @@
...
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN message sent to the partner for inbound AS2 transactions
+a|
+. In the *Transmission Flow* section, click *View MDN*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the CONTRL message sent to the partner
+a|
+. In the *Transmission Flow* section, click *View CONTRL*.
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: Please change as:
View and download the EDIFACT CONTRL or X12 997 acknowledgement message sent to the partner.

In the *Transmission Flow* section, click *View CONTRL* or *View 997*.

File: modules/ROOT/pages/activity-tracking.adoc [hunk 8]
```diff
@@ -1,87 +1,130 @@
...
+. In the *Sent transmission* field, click *View payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN received from the partner in response to outbound AS2 transmissions
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the *Received MDN* field, click *View payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+|View the CONTRL message received from the partner, if it exists
```
Comment (vijayang) [inline modules/ROOT/pages/activity-tracking.adoc]: View the EDIFACT CONTRL or X12 997 acknowledgement message received from the partner, if it exists

**Review by vijayang — CHANGES_REQUESTED** [review]

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/edifact-send-settings.adoc [hunk 9]
```diff
@@ -103,8 +105,11 @@ Carriage return and line feed
 Line Feed
 |CRLF
 |Test Indicator
-a| When you select *Test* from the drop-down, sets UNB-11 to `1`, which indicates that this is a test message.
-|Production
+a|Indicates if the EDIFACT transaction set is for testing or production. *Vijay, is this still true? There is no default that shows up.*
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/edifact-send-settings.adoc]: no answer from Vijay?

**Review by IsaacEldridge — CHANGES_REQUESTED** [review]
see comment

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**